README.md

```diff
@@ -1,2 +1,22 @@
 # zap-cli
-A Node.js CLI command for searching for, or opening a YouTube video with Zen Audio Player.
+
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zen-audio-player/zen-audio-player.github.io?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Build Status](https://travis-ci.org/zen-audio-player/zap-cli?branch=master)](https://travis-ci.org/zen-audio-player/zap-cli)
+[![GitHub Stats](https://img.shields.io/badge/github-stats-ff5500.svg)](http://githubstats.com/zen-audio-player/zap-cli)
+
+A Node.js CLI tool for searching for, or opening a YouTube video with [Zen Audio Player](https://github.com/zen-audio-player/zen-audio-player.github.io). To use the zap-cli command-line tool, clone the repository and run a global npm install inside of it.
```

Comment:
Still need the" usage" h2 after the first sentence here, see my previous comment